bitmap-team-awesome/lib/bit-file-helper.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const fs = require('fs');
+const Bitmap = require('../model/bitmap');
+
+module.exports = function(callback, callback2) {
+  fs.readFile(`${__dirname}/../../assets/bitmap.bmp`, function(err, data){
+    if (err) return callback(err);
+    var bitmap = data;
+    callback2 = new Bitmap(data);
```

Comment:
@meeksfred while `callback` is an appropriate naming convention in most scenarios, `callback2` should be avoided - that's the case for most things that you might inherently want to put a number on